client/src/http/BaseModel.ts

```diff
@@ -84,4 +84,10 @@ export default class BaseModel {
     const res = await http.post(path, data);
     return res.data.data;
   }
+
+  static async query(options: updateParamsType) {
```

Comment:
may be we can rename vectorSearch. They looks like same

Reply:
Has removed vectorSearch, use query instead.